eng/tools/select-packages/index.js

```diff
@@ -29,6 +29,7 @@ glob(filter, (err, files) => {
   }
 
   let packageTargets = "";
```

Comment:
Rename to `packageTargetsTo` for clarity and consistency

Reply:
done